packages/experimental/Expander/windows/ReactNativeExpander/ReactPackageProvider.cpp

```diff
@@ -13,7 +13,7 @@ namespace winrt::ReactNativeExpander::implementation
 
 void ReactPackageProvider::CreatePackage(IReactPackageBuilder const &packageBuilder) noexcept
 {
-    packageBuilder.AddViewManager(L"ExpanderViewManager", []() { return winrt::make<ExpanderViewManager>(); });
+    packageBuilder.AddViewManager(L"RNExpander", []() { return winrt::make<ExpanderViewManager>(); });
```

Comment:
making sure you updated the string in tsx as well, didn't see it in a quick scan over the files